### PR TITLE
Add logging and move list_keys module to list

### PR DESCRIPTION
**Pull request by mirons-google**
This includes logging to stdout and a dynamic logfile. This also renames the list_keys module to list to keep with the naming scheme of the other modules

### CODE REVIEW

File: keyrotator/create.py [hunk 1]
```diff
@@ -35,10 +35,11 @@ def run(self, project_id, iam_account, key_type=DEFAULT_KEY_TYPE,
     if output_file:
       self._write_key(path=output_file, payload=response)
     else:
-      print json.dumps(
-          response, sort_keys=True, indent=4, separators=(",", ": "))
+      logging.info("Key successfully created.")
+      loggin.debug("Key details: %s",
```
Comment (carise) [inline keyrotator/create.py]: loggin - typo?

**Review by carise — CHANGES_REQUESTED** [review]
Looks ok, is the test.log file necessary or can it be removed?

File: keyrotator/list_keys.py [hunk 2]
```diff
@@ -28,11 +30,8 @@ def run(self, project_id, iam_account):
     response = iam_service.list_keys(project_id, iam_account)
 
     if response and "keys" in response:
-      print "Key" + " " * 40 + "Created at" + " " * 13 + "Expires at"
-
       for key in response["keys"]:
         key_path = self.keyname_pattern.search(key["name"])
-        print "{}   {}   {}".format(
-            key_path.group(1), key["validAfterTime"], key["validBeforeTime"])
+        logging.info("Key: %s\n\tCreated: %s\n\tExpires: %s", key_path.group(1), key["validAfterTime"], key["validBeforeTime"])
```
Comment (carise) [inline keyrotator/list_keys.py]: Could you change so the line length is 80 chars?

**Comment by mirons-google** [comment]
Fixed. PTAL.

File: keyrotator/list.py [hunk 3]
```diff
@@ -12,27 +12,26 @@
...
     response = iam_service.list_keys(project_id, iam_account)
 
     if response and "keys" in response:
-      print "Key" + " " * 40 + "Created at" + " " * 13 + "Expires at"
-
       for key in response["keys"]:
         key_path = self.keyname_pattern.search(key["name"])
-        print "{}   {}   {}".format(
-            key_path.group(1), key["validAfterTime"], key["validBeforeTime"])
+        logging.info("Key: %s\n\tCreated: %s\n\tExpires: %s", key_path.group(1), key["validAfterTime"], key["validBeforeTime"])
```
Comment (carise) [inline keyrotator/list.py]: Line length should be <= 80 chars

**Review by carise — APPROVED** [review]
LGTM

**Pull request merged**